### ISSUE TITLE
About Linear Probe Accuracy of Resenet-50

### ISSUE THREAD

**Issue by avinabsaha** [facebookresearch/moco-v3#40]
I ran the code using the parameters specified in the CONFIG.md for Resnet-50, 1000 epoch pre-training and then fine-tuned using the linear probe method. All the parameters were kept the same as mentioned in the CONFIG.md file. However, after linear probing, my accuracy can reach only 74.36% and not 74.6%.  I am not sure what I might be missing here. 

Could you help me out?

Additionally, I ran the official checkpoint provided and that is able to achieve 74.6%.
